tests/test_manifests_job.py

```diff
@@ -48,122 +58,148 @@ def manifest_content(self, eco):
         self.output = MockDB().run(qry)
         self.job_id = job_id
         self.state = 'PENDING'
 
     def result(self):
+        """Get the output of input query."""
         return self.output
 
     def done(self):
+        """Check if job run was complete."""
         return True
 
     def __iter__(self):
+        """Return job output as an iterable."""
         return iter(self.output)
 
 
 class MockBigQuery(mock.Mock):
+    """Mocks Google's Big Query Runner."""
 
     QueryJobConfig = type('QueryJobConfig', (), {'priority': None})
 
     def __init__(self, *args, **kwargs):
+        """Initialize the MockBigQuery object."""
         super().__init__(*args, **kwargs)
         self.job = self.Job
 
     class Client:
+        """Mocks the Google Big Query Client."""
 
         def __init__(self, *args, **kwargs):
+            """Initialize the Client object."""
             self._state_flag = False
 
         def query(self, qry, *args, **kwargs):
+            """Create and execute a manifest retrieval query."""
             self.qry = qry
             return QueryJob(self.qry)
 
         def get_job(self, job_id):
+            """Get job status details."""
             query_job = QueryJob(qry=self.qry, job_id=job_id)
             query_job.state = ['PENDING', 'DONE'][self._state_flag]
             self._state_flag = True
             return query_job
 
     class Job:
+        """Mocks Big Query Jobs."""
+
         QueryJobConfig = type('QueryJobConfig', (), {})
 
     class QueryPriority:
+        """Priority aspect of queries."""
+
         BATCH = 'batch'
 
 
 class MockS3(LocalDataStore):
+    """Mocks AWS S3 storage."""
 
     def __init__(self, *args, **kwargs):
+        """Initialize the MockS3 object."""
         super().__init__(*args, **kwargs)
         self.is_connected = lambda: True
+        self.bucket_name = 'developer-analytics-audit-report'
 
     def object_exists(self, fname):
+        """Check if file exists in S3."""
         return pathlib.Path(self.src_dir).joinpath(fname).exists()
 
     def write_json_file(self, fname, content):
+        """Dump manifest json data to S3."""
         fpath = pathlib.Path(self.src_dir).joinpath(fname)
         if not fpath.parent.exists():
             fpath.parent.mkdir(parents=True, exist_ok=True)
         with open(str(fpath.absolute()), 'w') as json_fileobj:
             return json.dump(content, json_fileobj)
 
     def connect(self):
+        """Connect to AWS S3 instance."""
         return self.is_connected()
 
     def __del__(self):
+        """Delete S3 directory."""
         shutil.rmtree(self.src_dir)
 
 
 @pytest.fixture
 @mock.patch('rudra.data_store.bigquery.base.bigquery',
             new_callable=MockBigQuery)
 def _data_process_client_maven(_mock_bigquery_obj):
-    from rudra.data_store.bigquery.maven_bigquery import MavenBigQuery, MavenBQDataProcessing
-    _mvn_ins = MavenBigQuery()
+    _mvn_ins = manifests_job.mvn_bq.MavenBigQuery()
```

Comment:
The docstyle run will fail without the docstrings for the test functions. This comment holds good for any function without a docstring.

Reply:
I think it didn't throw up an error when I ran it because maybe it's a fixture @samuzzal-choudhury ? But I will add in docstrings to this.